vector_indexers/plot.py

```diff
@@ -0,0 +1,68 @@
+import click
+import matplotlib.pyplot as plt
+import json
+import pandas as pd
+
+
+def set_k(recall_at):
+    if recall_at is 1:
+        k = 0
+    elif recall_at is 10:
+        k = 1
+    elif recall_at is 20:
+        k = 2
+    elif recall_at is 50:
+        k = 3
+    elif recall_at is 100:
+        k = 4
+    else:
+        print("K value not valid. It should be either 1, 10, 20, 50 or 100. K = 10 will be used")
+        k = 1
+    return k
+
+
+@click.command()
+@click.option('--recall_at', '-k', default=10)
+def create_plot(recall_at):
+    k = set_k(recall_at)
+
+    input_file = open('results.json')
+    json_array = json.load(input_file)
+
+    result_list = []
+    recall_list = []
+
+    for item in json_array:
+        result_details = {"index_time": None,
```

Comment:
Use constants from app.py (Put them in a separate file and import them both from `app.py` and `plot.py`